templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -253,6 +253,10 @@ contents:
             clone_mac=0
           fi
         fi
+      elif [ $(nmcli --get-values connection.type conn show ${old_conn}) == "tun" ]; then
```

Comment:
I would follow the same style as the ifs above and quote the nmcli command substitution

Reply:
@jcaamano updated.